theano/compile/mode.py

```diff
@@ -65,13 +65,26 @@ def register_linker(name, linker):
 OPT_FAST_COMPILE.name = 'OPT_FAST_COMPILE'
 OPT_STABILIZE.name = 'OPT_STABILIZE'
 
+OPT_O2 = OPT_FAST_COMPILE.including('fusion')
+OPT_O3 = OPT_FAST_RUN.excluding('inplace')
+OPT_UNSAFE = OPT_O3.including('unsafe')
```

Comment:
Are you sure? Faruk made this based on your comment https://github.com/Theano/Theano/issues/5762#issuecomment-289162140 
> unsafe= (o3 + opt_inclusing=unsafe)
Or you meant o4 because you were previously defining them as o0, ... ,o3? You use both namings in the comment, o0 to 03 and o1 to o4.